cypress-tests/README.md

```diff
@@ -34,65 +81,68 @@ To run test cases, follow these steps:
    npm ci
    ```
 
-3. Insert data to `cards_info` table in `hyperswitch_db`
+   Once installed, verify the installation by running:
 
    ```shell
-   psql --host=localhost --port=5432 --username=db_user --dbname=hyperswitch_db --command "\copy cards_info FROM '.github/data/cards_info.csv' DELIMITER ',' CSV HEADER;"
+    npx cypress --version
    ```
 
-4. Set environment variables for cypress
+   To learn about the supported commands, execute:
 
    ```shell
-   export CYPRESS_CONNECTOR="connector_id"
-   export CYPRESS_BASEURL="base_url"
-   export DEBUG=cypress:cli
-   export CYPRESS_ADMINAPIKEY="admin_api_key"
-   export CYPRESS_CONNECTOR_AUTH_FILE_PATH="path/to/creds.json"
+   npm run
    ```
 
-5. Run Cypress test cases
-
-   To run the tests in interactive mode run the following command
+3. Set up the cards database:
 
    ```shell
-   npm run cypress
+   psql --host=localhost --port=5432 --username=db_user --dbname=hyperswitch_db --command "\copy cards_info FROM '.github/data/cards_info.csv' DELIMITER ',' CSV HEADER;"
    ```
 
-   To run all the tests in headless mode run the following command
+4. Set environment variables for cypress
 
    ```shell
-   npm run cypress:ci
+   export CYPRESS_CONNECTOR="connector_id"
+   export CYPRESS_BASEURL="base_url"
+   export DEBUG=cypress:cli
+   export CYPRESS_ADMINAPIKEY="admin_api_key"
+   export CYPRESS_CONNECTOR_AUTH_FILE_PATH="path/to/creds.json"
    ```
 
-   To run payment tests in headless mode run the following command
+> [!TIP]
+> It is recommended to install [direnv](https://github.com/direnv/direnv) and use a `.envrc` file to store these environment variables with `cypress-tests` directory. This will make it easier to manage environment variables while working with Cypress tests.
 
-   ```shell
-   npm run cypress:payments
-   ```
+> [!NOTE]
+> To learn about how `creds` file should be structured, refer to the [example.creds.json](#example-credsjson) section below.
 
-   To run payout tests in headless mode run the following command
+## Running Tests
 
-   ```shell
-   npm run cypress:payouts
-   ```
+Execution of Cypress tests can be done in two modes: Development mode (Interactive) and CI mode (Headless). The tests can be executed against a single connector or multiple connectors in parallel. Time taken to execute the tests will vary based on the number of connectors and the number of tests. For a single connector, the tests will take approximately 07-12 minutes to execute.
```

Comment:
> For a single connector, the tests will take approximately 07-12 minutes to execute.

This could also vary with hardware configuration though.

Reply:
Unfortunately, yes. I wanted to let the user know about the usual time taken for the test to run. I'll a notification in the brackets about hardware configuration.